### PR TITLE
Add internationalization of Chinese language

### PR DESCRIPTION
**Pull request by bingxue314159**

### CODE REVIEW

**Comment by Tostino** [comment]
Thanks for the PR. Will look at it shortly and see about merging it.

**Comment by Tostino** [comment]
@bingxue314159 Just wondering what dialect of Chinese this is? I really don't know much about the different dialects, so I thought it better to get more info prior to merging.

Sorry for the extreme delay!

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Password #2, while not allowed by our policy, is only susceptible to a brute for
         - Spanish (es)
         - Telugu (te)
         - Ukrainian (uk)
+        - Chinaese (zh)
```
Comment (Tostino) [inline README.md]: This seems to be misspelled.

Comment (bingxue314159) [reply README.md]: Sorry, it is indeed a spelling mistake. It should be Chinese.

**Pull request merged**